assets/javascripts/discourse-assign/controllers/group-assignments.js.es6

```diff
@@ -1,31 +1,33 @@
 import { inject as service } from "@ember/service";
 import Controller, { inject as controller } from "@ember/controller";
+import { ajax } from "discourse/lib/ajax";
 
 export default Controller.extend({
   router: service(),
   application: controller(),
   loading: false,
+  offset: 0,
 
   findMembers(refresh) {
-    if (this.loading || !this.model) {
+    if (refresh) {
+      this.set("members", this.model.members);
```

Comment:
Do we need this? I don't think that `model.members` will have the assignment count? So maybe we should set to an empty array?

Reply:
`model.members` have assignment count, we need to set it here as when the `setupController` will be called, `members` will be empty. To set it to members from the model, we need to do this

Reply:
Aha! I see now. I thought `model` was a `group` model, but it's not.

Makes sense :+1: